### PR TITLE
New docs review

### PR DESCRIPTION
**Pull request by raddevon**
I think I'm too in the weeds here. I was trying to go through everything as part of my review of the docs and fix every problem I found which I don't think is practical right now. I'd like to get merged the things I did find though, and we can return to this later. I'll take a broader view for my review going forward.

This also prompted some changes in the web site repo: https://github.com/edgedb/edgedb.com/pull/822

### CODE REVIEW

File: docs/intro/quickstart.rst [hunk 1]
```diff
@@ -471,10 +477,9 @@ and used a client library.
...
-  `Python </docs/clients/00_python/index>`__, and
-  `Go </docs/clients/02_go/index>`__.
-
-- Or just jump into the :ref:`docs <index_toplevel>`!
+- To start building an application using the language of your choice, check out
+  our client libraries: :ref:`JavaScript/TypeScript <edgedb-js-intro>`, `Go
+  </libraries/go>`__, :ref:`Python <edgedb-python-intro>`, :ref:`Rust
+  <ref_rust_index>`, `C# and F# </libraries/dotnet>`__, :ref:`Java
+  <edgedb-java-intro>`, `Dart </libraries/dart>`__, and :ref:`Elixir
+  <edgedb-elixir-intro>`.
```
Comment (beerose) [inline docs/intro/quickstart.rst:485]: Nitpicking here, but maybe we can make it a bullet list as we have more of them now?

File: docs/intro/quickstart.rst [hunk 2]
```diff
@@ -436,11 +437,15 @@ will look something like this:
...
-`Rust <https://github.com/edgedb/edgedb-rust>`__, and
-`C# and F# <https://github.com/edgedb/edgedb-net>`_.
+:ref:`JavaScript/TypeScript <edgedb-js-intro>`,
+`Go </libraries/go>`__,
+:ref:`Python <edgedb-python-intro>`,
+:ref:`Rust <ref_rust_index>`,
+`C# and F# </libraries/dotnet>`__,
+:ref:`Java <edgedb-java-intro>`,
+`Dart </libraries/dart>`__, and
+:ref:`Elixir <edgedb-elixir-intro>`.
```
Comment (beerose) [inline docs/intro/quickstart.rst:447]: Same here.

File: docs/guides/datamigrations/postgres.rst [hunk 3]
```diff
@@ -146,7 +146,7 @@ for now. Incidentally, even if the ``id`` was specified as a ``uuid`` value
 the recommended process is to record it as ``app_id`` as opposed to try and
 replicate it as the main object ``id``. It is, however, also possible to bring
 it over as the main ``id`` by adjusting certain :ref:`client connection
-<ref_std_cfg_client_connections>` settings. The column ``client_settings``
+settings <ref_std_cfg_client_connections>` . The column ``client_settings``
```
Comment (beerose) [inline docs/guides/datamigrations/postgres.rst:149]: Is the space before `.` on purpose?
```suggestion
settings <ref_std_cfg_client_connections>`. The column ``client_settings``
```

File: docs/intro/quickstart.rst (hunk 1 above)
Comment (raddevon) [reply docs/intro/quickstart.rst]: Good call. Formatted both this and the other one as proper lists.

File: docs/guides/datamigrations/postgres.rst (hunk 3 above)
Comment (raddevon) [reply docs/guides/datamigrations/postgres.rst]: Not intentional. Thank you for catching it!

**Review by beerose — APPROVED** [review]

**Pull request merged**